charts/camunda-platform/charts/identity/templates/_helpers.tpl

```diff
@@ -242,3 +242,41 @@ https://docs.bitnami.com/kubernetes/apps/keycloak/configuration/manage-passwords
     admin-password
 {{- end }}
 {{- end -}}
+
+{{/*
+[identity] PostgreSQL helpers.
+*/}}
+
+{{- define "identity.postgresql.id" -}}
+    {{- (printf "%s-%s" .Release.Name .Values.postgresql.nameOverride) | trunc 63 | trimSuffix "-" }}
+{{- end -}}
+
+{{- define "identity.postgresql.secretName" -}}
+    {{- $autSecretName := .Values.postgresql.auth.existingSecret | default (include "identity.postgresql.id" .) -}}
+    {{- .Values.externalDatabase.enabled | ternary .Values.externalDatabase.existingSecret $autSecretName }}
+{{- end -}}
+
+{{- define "identity.postgresql.secretKey" -}}
+    {{- .Values.externalDatabase.enabled | ternary .Values.externalDatabase.existingSecretPasswordKey "postgres-password" }}
```

Comment:
`postgres-password` relates to the `postgres` user, the value for the user we provide is stored in the `password` key
```suggestion
    {{- .Values.externalDatabase.enabled | ternary .Values.externalDatabase.existingSecretPasswordKey (print "password") }}
```

Reply:
Good point :ok_hand: 

But in that case, it shouldn't be then `identity-password` so it's clear that it's used for Identity (following the same pattern of other components)

Reply:
Either or works for me :)